.rubocop.yml

```diff
@@ -7,6 +7,12 @@ AllCops:
   Exclude:
     - db/schema.rb
 
+
+Gemspec/RequiredRubyVersion:
+  Exclude:
+    - modules/**/*.gemspec
```

Comment:
IMO: inline module "gems" should not define their own ruby version and instead rely on the main definition

Reply:
:+1: